template/src/utils/SdkEvents.ts

```diff
@@ -34,6 +34,19 @@ export interface userEventsMapInterface {
   'rtc-user-unpublished': (uid: UidType, trackType: 'audio' | 'video') => void;
   'rtc-user-joined': (uid: UidType) => void;
   'rtc-user-left': (uid: UidType) => void;
+  '_rtm-joined': (uid: UidType) => void;
+  'devices-selected-microphone-changed': (
```

Comment:
'devices' is a kind of tag so that all device events share a root. selected was added to be more verbose and indicate the selected microphone was changed not the device list. But happy to change if not deemed necessary